apps/framework-cli/src/infrastructure/stream/redpanda.rs

```diff
@@ -160,6 +160,8 @@ pub fn create_subscriber(config: &RedpandaConfig, group_id: &str, topic: &str) -
     client_config
         .set("session.timeout.ms", "6000")
         .set("enable.partition.eof", "false")
+        .set("enable.auto.commit", "true")
+        .set("auto.commit.interval.ms", "1000")
```

Comment:
Do these values need to be set via environment variables for cloud use?

Reply:
Hum no - I believe they would stay the same, the 1000 value is synced with the `flush` frequency on the clickhouse client side